runtime/vm/resolvesupport.cpp

```diff
@@ -52,7 +52,7 @@ static bool finalFieldSetAllowed(J9VMThread *currentThread, bool isStatic, J9Met
 * @return true if access is legal, false if not
 */
 static bool
-finalFieldSetAllowed(J9VMThread *currentThread, bool isStatic, J9Method *method, J9Class *fieldClass, J9ROMFieldShape *field, UDATA jitFlags)
+finalFieldSetAllowed(J9VMThread *currentThread, bool isStatic, J9Method *method, J9Class *fieldClass, J9Class *methodClass, J9ROMFieldShape *field, UDATA jitFlags)
```

Comment:
`methodClass` needs to be added to the `@param` section of the function description

Reply:
`methodClass` -> `callerClass` or `setterClass` or .... It might be clearer to call this something other than `methodClass`.  

I had to do a double take on the `(fieldClass != methodClass)` check below purely due to the naming.